apps/api/src/app/organization/usecases/get-In-app-activated/get-In-app-activated.usecase.ts

```diff
@@ -0,0 +1,18 @@
+import { SubscriberRepository } from '@novu/dal';
+import { GetInAppActivatedCommand } from './get-In-app-activated.command';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class GetInAppActivated {
+  constructor(private readonly subscriberRepository: SubscriberRepository) {}
+
+  async execute(command: GetInAppActivatedCommand): Promise<boolean> {
+    const inAppSubscriberCount = await this.subscriberRepository.find({
+      _organizationId: command.organizationId,
+      isOnline: { $exists: true },
+      subscriberId: { $not: /cli-/ },
```

Comment:
This part is good because I added additional code on the CLI package to the subscriber id, **but** it will be updated to equal to the one from snippets on the web client of the subscriber id (which will be reflated here)